h/presenters/group_json.py

```diff
@@ -6,9 +6,9 @@
 class GroupJSONPresenter(object):
     """Present a group in the JSON format returned by API requests."""
 
-    def __init__(self, group, route_url=None):
+    def __init__(self, group, link_svc=None):
```

Comment:
Calling this param `link_svc` may be not good; it's a callable...a method _on_ a service but not an actual service.